src/transforms/crop.jl

```diff
@@ -57,17 +68,16 @@ function _crop(box::Box{<:𝔼}, limits)
   lims = _xyzlimits(limits)
   min = convert(Cartesian, coords(minimum(box)))
   max = convert(Cartesian, coords(maximum(box)))
-  xyzmin, xyzmax = _xyzminmax(min, max, lims)
-  Box(withcrs(box, xyzmin), withcrs(box, xyzmax))
+  _xyzminmax(min, max, lims)
 end
 
 function _crop(box::Box{🌐}, limits)
```

Comment:
the function _crop is computing adjusted limits so it could be called _fixlimits or more descriptive name instead